src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql

```diff
@@ -33,6 +33,7 @@ WHERE
       "ICD10EXP",
       "ICD10WHO",
       "ICD11",
+      "icd11.foundation",
```

Comment:
## `icd11.foundation` prefix
Yep, this is the prefix that we are using for this source!